packages/streams/simple.js

```diff
@@ -167,7 +167,16 @@ const dataTypeMapping = {
     return result.concat([new N2kToSignalK(options.subOptions)])
   },
   NMEA2000YD: options => {
-    const result = [new Ydwg02(options.subOptions)]
+    let subOptions
+    if ( options.subOptions.type === 'ydwg02-usb-canboatjs' )
+    {
+      subOptions = {...options.subOptions, usb: true}
```

Comment:
Ydwg02 is already getting subOptions.type, why does the  `usb` need to be populated here?

Reply:
I don't want caboatjs to have know about `ydwg02-usb-canboatjs`